### ISSUE TITLE
The color palette for the entire site fails aria color contrast standards

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7182]
### Where To Find Bug
All pages

### Expected Behaviour
Text under 18 point regular font or 14 point bold font must have a contrast ratio of at least 4.5 to 1 against the background.

### Actual Behaviour
All of the color combinations (foreground, background) for the entire site do not meet aria color contrast standards except for the text with color #575E75 (foreground on white background).

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Notice the color palette used for the site

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
